### DISCUSSION TITLE
clarification on 'pendingShares' number and why is it so high

### DISCUSSION THREAD

**Discussion by mboyd1** [coinfoundry/miningcore#1710] [General]
the miningcore api reports very large numbers in the billions for `pendingShares` when querying /pool/[poolid]/miners/[address]

for example for this miner it says there are 42,515,513,500 pending shares.   
![Screenshot 2023-08-01 at 8 56 55 AM](https://github.com/oliverw/miningcore/assets/5233780/344be2bf-791e-4edd-8017-b2bf6c88cee2)

I am struggling to understand why this number is so high, the asics only submit shares every 15 seconds or something there seems to be no way possible it has submitted 42 billion unique proof-of-work stratum shares.  Is this number some multiple of the shares and their difficulty, or is there a decimal place that is way off somehow?

**Comment by MiningCryptoLive** [comment]
That number is the shares total difficulty
